src/main/groovy/nebula/plugin/plugin/NebulaPluginPlugin.groovy

```diff
@@ -44,7 +48,7 @@ class NebulaPluginPlugin implements Plugin<Project> {
                                       'nebula.maven-publish',
                                       'nebula.publish-verification',
                                       'nebula.nebula-release',
-                                      'nebula.nebula-bintray', // nebula-bintray needs to happened after nebula-release since version isn't lazy in the bintray extension
+                                      'nebula.oss-publishing', // nebula.oss-publishing  needs to happened after nebula-release since version isn't lazy in the bintray extension
```

Comment:
If the comment is the same, it should read:
```
nebula.oss-publishing  needs to happened after nebula-release since version isn't lazy in the oss-publishing extension
```